src/core/CONVENTIONS.md

```diff
@@ -1,15 +1,35 @@
-# Kibana Conventions
-
-- [Kibana Conventions](#kibana-conventions)
+- [Organisational Conventions](#organisational-conventions)
+  - [Definition of done](#definition-of-done)
+- [Technical Conventions](#technical-conventions)
   - [Plugin Structure](#plugin-structure)
     - [The PluginInitializer](#the-plugininitializer)
     - [The Plugin class](#the-plugin-class)
     - [Applications](#applications)
     - [Services](#services)
     - [Usage Collection](#usage-collection)
     - [Saved Objects Types](#saved-objects-types)
-
-## Plugin Structure
+  - [Naming conventions](#naming-conventions)
+
+## Organisational Conventions
+### Definition of done
+Definition of done for a feature:
+- has a dedicated Github issue describing problem space
```

Comment:
Isn't that more in the 'Definition of Ready'?

Reply:
Agree, do we want to have a separate document? We can keep it here for the time being.

Reply:
Keeping it there for now is fine.